SPECS/selinux-policy/selinux-policy.spec

```diff
@@ -311,6 +332,23 @@ exit 0
 selinuxenabled && semodule -nB
 exit 0
 %changelog
+* Mon May 23 2022 Chris PeBenito <chpebeni@microsoft.com> - 2.20220106-7
+- Fix previous multipath LVM changes.
+- Add types for devices.
+- Cherry pick upstream commit for container fds.
+- Allow container engines to connect to http cache ports.
+- Allow container engines to stat() generic (device_t) devices.
+
+* Mon May 02 2022 Chris PeBenito <chpebeni@microsoft.com> - 2.20220106-6
+- Additional compatibility for Fedora container-selinux.
+- Remove unneeded systemd_run_t domain
+- Updates for multipath LVM
+- Fix for console logins
+- New type for SAS management devices
+
+* Fri Apr 29 2022 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.20220106-5
+- Fixing source URL.
```

Comment:
I think this change didn't get applied. Could you replace `$` in `Source0` with `%`? That's all my change did here, I believe.